### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jafacakes2011**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -113,6 +113,9 @@ pytest:
 test:
 	$(run) leavers pytest --disable-warnings --reuse-db $(test)
 
+cam:
+	$(run) leavers pytest --disable-warnings --reuse-db leavers/tests/views/test_line_manager.py::TestLineManagerAccessMixin::test_user_has_permission
+
```
Comment (SamDudley) [inline makefile:118]:
```
make test test=leavers/tests/views/test_line_manager.py::TestLineManagerAccessMixin::test_user_has_permission
```

File: core/context_processors.py [hunk 2]
```diff
@@ -63,6 +63,13 @@ def global_context(request):
                     reverse("list-assets"),
                 )
             )
+        if "HR" in user_group_names:
```
Comment (SamDudley) [inline core/context_processors.py:66]: Should we check a permission here and try to move away from checking a group?

File: leavers/views/common.py [hunk 3]
```diff
@@ -15,8 +23,20 @@ class LeavingRequestListView(UserPassesTestMixin, LeavingRequestListing):
...
+
+class LeavingRequestView(BaseTemplateView, LeavingRequestViewMixin):
+    template_name = "leaving/common/leaving_request.html"
+    back_link_url = reverse_lazy("leaving-requests-list")
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        leaver_name = self.leaving_request.get_leaver_name()
+        possessive_leaver_name = make_possessive(leaver_name)
```
Comment (SamDudley) [inline leavers/views/common.py:38]: I think I added a template tag for this.

**Review by SamDudley — APPROVED** [review]

File: makefile (hunk 1 above)
Comment (jafacakes2011) [reply makefile:118]: ahh i couldn't work that out
didn't mean to commit this tho!

File: core/context_processors.py (hunk 2 above)
Comment (jafacakes2011) [reply core/context_processors.py:66]: 100% in the future!

File: leavers/views/common.py (hunk 3 above)
Comment (jafacakes2011) [reply leavers/views/common.py:38]: Yea, but we need to do it here for the page_title context

**Pull request merged**